estate/models/estate_property.py

```diff
@@ -0,0 +1,86 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from datetime import date, timedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "An Estate Property"
+    _order = "id DESC"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=date.today() + timedelta(days=90))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        [
+            ('north', 'North'),
+            ('west', 'West'),
+            ('east', 'East'),
+            ('south', 'South')
```

Comment:
Try to use single quote `'` for strings that will not be displayed in the UI, i.e. dict keys and double quote `"` for UI strings 
```suggestion
            ('north', "North"),
            ('west', "West"),
            ('east', "East"),
            ('south', "South"),
```

plus, add the comma at the end :+1: